File: python/VowelConso.py
ch = input("Enter a Alphabetical Character: ").lower()[0] #change to lower case at index 0
vowel = False
'''
    *  note: java hasn't any method insert character so we use charAt()
    * charAt() return a character at index "n" of string
'''

if(ch == 'a' or ch == 'e' or ch == 'i' or ch == 'o' or ch == 'u'):
    vowel = True
else:
    vowel = False

if(vowel):
    print("Vowel Character")
else:
    print("Consonent Character")

File: c++/VowelConso.cpp
#include <iostream>
using namespace std;

int main()
{
    char ch;
    int isVowel = 0;
    cout << "Enter a Alphabetical Character ";
    cin >> ch;
    

    if ((ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u')||
        (ch == 'A' || ch == 'E' || ch == 'I' || ch == 'O' || ch == 'U'))
    { 
        isVowel = 1;
    }

    if (isVowel)
    {
        cout << " is Vowel Character" << ch;
    }
    else
    {
        cout << "is Consonent Character" << ch;
    }
}